Update controlling_concurrency.md (#1919)
buildkite/docs: pages/pipelines/controlling_concurrency.md

## Concurrency and parallelism

Sometimes you need strict concurrency while also having jobs that would benefit from parallelism.

> In these situations you can use *concurrency gates* to control which jobs run in parallel and which jobs run one at a time. Concurrency gates come in pairs, for it to work, when you open a concurrency gate, you have to close it.

Reviewer: I think the sentence is a bit too complex. How about removing one of the comma sections (and add a missing comma elsewhere):

```suggestion
In these situations, you can use *concurrency gates* to control which jobs run in parallel and which jobs run one at a time. Concurrency gates come in pairs, so when you open a gate, you have to close it.
```

What do you think? We could also split it into multiple sentences.

Author: @mbelton-buildkite Yes we definitely can
